examples/asr/asr_chunked_inference/ctc/speech_to_text_buffered_infer_ctc.py

```diff
@@ -89,6 +94,12 @@ class TranscriptionConfig:
     # Recompute model transcription, even if the output folder exists with scores.
     overwrite_transcripts: bool = True
 
+    # Config for word / character error rate calculation
+    calculate_wer: bool = True
+    clean_groundtruth_text: bool = False
```

Comment:
Would you please add some comments on the top for users how and when to use these two arguments: clean_groundtruth_text and langid?